bioio/writers/generate_zarr.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
This file won't go in the repo but is cool to play around with to make your own zarrs